app/assets/stylesheets/spree/frontend/components/layout/_header.scss

```diff
@@ -16,7 +16,46 @@
     &__content {
       align-items: center;
       display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
       padding: 2rem 3rem;
+
+      @include desktop {
+        flex-wrap: nowrap;
+        justify-content: flex-end;
+      }
+
+      .logo {
+        order: 1;
+
+        img {
+          width: 10em;
+
+          @include tablet {
+            width: auto;
+          }
+        }
+      }
+
+      .search-bar {
```

Comment:
We should move this styles to `_search_bar.scss`. Is there a reason why they are here?